bitcoin/chainparams.c

```diff
@@ -53,6 +53,16 @@ const struct chainparams networks[] = {
      .cli_args = "-testnet",
      .dust_limit = 100000,
      .when_lightning_became_cool = 1,
+     .testnet = true},
+      {.index = 5,
```

Comment:
NACK, don't need to add even more networks, I'd remove litecoin-testnet as well, testing should always be done on bitcoin testnet and regtest.

Reply:
regtest for LTC is for development and easy test essential
in litecoin rltc is the bech32 tag

Reply:
I think regtest for litecoin should be supported:

1. it does not cost too much to add.
2. If I want to build something based on clightning for making atomic swap on lightning, I need at least 2 regtest.